examples/solids/index.rst

```diff
@@ -338,16 +338,35 @@ Integrating by parts, we arrive at the weak form:
 where :math:`\boldsymbol{P} \cdot \hat{\boldsymbol{N}}` is a prescribed boundary
 condition written in terms of the reference configuration.
 
-For simplicity, we express here the constitutive law in indicial notation:
+Equation :math:numref:`1st2nd` devises a method to model a hyperelastic solid with a Neo-Hookean constitutive law. Evaluatinig the derivative of this model, yields a forth order tensor:
+
+.. math::
+   :label: mtfs
+
+   \dfrac{\partial \bm P}{\partial \bm F} = \bm S + \left[ \lambda \bm F^{-1} \bm F^{-1} \left(\lambda \log(J) - \mu \right) \left(\bm F^{-1}\bm F^{-1} + \bm C^{-1} \right) \right]
```

Comment:
FWIW, I prefer to write linearizations in delta form. That is, I'd write `\frac{\partial P}{\partial F} \delta F` where `\delta F` becomes the Newton step you're solving for. For the sake of interpretability, it can be useful to split terms,
```
\frac{\partial P}{\partial F} \delta F = \delta F S + F \frac{\partial S}{\partial E} \delta E
```
where `\delta E = \frac{\partial E}{\partial F} \delta F` and then you can focus on writing down `\frac{\partial S}{\partial E} \delta E`, which is the important interpretable part. This is also the way I would write the code, BTW. (I expect there will be interest in adding different hyperelastic models, so factoring the code well is important.)